tests/cases/expr/mod.slint

```diff
@@ -23,6 +25,7 @@ assert_eq(instance.get_t4(),432);
 assert_eq(instance.get_t5(), 1.8);
 assert_eq(instance.get_t6(), 2.);
 assert_eq(instance.get_t7(), 3.5);
+assert_eq(instance.get_t8(), 0);
```

Comment:
I also tried adding a `assert(instance.get_test());` but that failed because t5 is 1.7999992 which differ from 1.8 by `7.2e-7` while the epsilon margin is `1.19e-7`

I didn't want to bother with that for now.